vendor/src/github.com/docker/engine-api/types/types.go

```diff
@@ -509,3 +510,18 @@ type Runtime struct {
 	Path string   `json:"path"`
 	Args []string `json:"runtimeArgs,omitempty"`
 }
+
+type ContainerStats struct {
+	Name             string
+	CPUPercentage    float64
+	Memory           float64
+	MemoryLimit      float64
+	MemoryPercentage float64
+	NetworkRx        float64
+	NetworkTx        float64
+	BlockRead        float64
+	BlockWrite       float64
+	PidsCurrent      uint64
+	Mu               sync.Mutex
+	Err              error
```

Comment:
Not sure we should move these to engine-api just to get formatting on the client.